2.x feature 2.x develop | Updated: session keys in env

we-vikram-wri231 opened the pull request:
https://img-v1.dev.getdemo.dev/screenshot/chrome_8IdsdPQYaC.mp4

we-alam-a001 committed:
Updated: language strings and categories

we-alam-a001 committed:
Updated: language strings and categories

we-alam-a001 committed:
Updated: language strings and categories

we-alam-a001 committed:
Updated: language strings and categories

we-alam-a001 committed:
Updated: localization in pages and store of users

we-alam-a001 committed:
Updated: localization in pages and store of users

we-alam-a001 committed:
Updated: localization in pages and store of users

we-alam-a001 committed:
Updated: localization in pages and store of users

we-alam-a001 committed:
Updated: localization of sidebar menu

we-alam-a001 committed:
Updated: localization of sidebar menu

we-alam-a001 committed:
Updated: localization of sidebar menu

we-alam-a001 committed:
Added: search field in localization page

we-alam-a001 committed:
Added: search field in localization page

we-alam-a001 committed:
Added: search field in localization page

we-alam-a001 committed:
Updated: seeds of response in users page

we-alam-a001 committed:
Updated: seeds of response in users page

we-alam-a001 committed:
Updated: seeds of response in users page

we-alam-a001 committed:
Updated: seeds of response in users page

we-alam-a001 committed:
Updated: seeds in controller and model of users

we-alam-a001 committed:
Updated: seeds in controller and model of users

we-alam-a001 committed:
Updated: seeds in controller and model of users

we-alam-a001 committed:
Updated: seeds in controller and model of users

we-alam-a001 committed:
Updated: validation string of users

we-alam-a001 committed:
Updated: validation string of users

we-alam-a001 committed:
Updated: validation string of users

we-alam-a001 committed:
Updated: validation string of users

we-alam-a001 committed:
Updated: all response from users page

we-alam-a001 committed:
Updated: all response from users page

we-alam-a001 committed:
Updated: all response from users page

we-alam-a001 committed:
Updated: all response from users page

we-alam-a001 committed:
Merge remote-tracking branch 'origin/2.x-feature/update-users-localization' into 2.x-feature/update-users-localization

we-alam-a001 committed:
Fixed: error in users store

we-alam-a001 committed:
Fixed: error in users store

we-alam-a001 committed:
Updated: label of add custom field link

we-alam-a001 committed:
Updated: label of add custom field link

we-alam-a001 committed:
Updated: users language string

we-alam-a001 committed:
Updated: users language string

we-vikram-wri231 committed:
Updated: setup configuration and account page

we-vikram-wri231 committed:
Updated: setup configuration and account page

we-vikram-wri231 committed:
Updated: setup configuration and account page

we-vikram-wri231 committed:
Updated: setup configuration and account page

we-vikram-wri231 committed:
Updated: env app name to session prefix and session cookie

we-vikram-wri231 committed:
Updated: env app name to session prefix and session cookie

we-vikram-wri231 committed:
Updated: env app name to session prefix and session cookie

we-vikram-wri231 committed:
Updated: rebuild assets

we-vikram-wri231 committed:
Updated: rebuild assets

we-vikram-wri231 committed:
Updated: rebuild assets

we-vikram-wri231 committed:
Updated: rebuild assets

we-vikram-wri231 committed:
Updated: restricted action

we-vikram-wri231 committed:
Updated: restricted action

we-vikram-wri231 committed:
Updated: restricted action in bulk action

we-vikram-wri231 committed:
Updated: restricted action in bulk action

we-vikram-wri231 committed:
Updated: restricted action

we-vikram-wri231 committed:
Updated: restricted action

we-vikram-wri231 committed:
Fixed: permission seeds of taxonomy section

we-vikram-wri231 committed:
Fixed: permission seeds of taxonomy section

we-vikram-wri231 committed:
Updated: restricted action for bulk action

we-vikram-wri231 committed:
Updated: restricted action for bulk action

we-vikram-wri231 committed:
Updated: Active all logic

we-vikram-wri231 committed:
Updated: Active all logic

we-vikram-wri231 committed:
Fixed: role permission active all logic

we-vikram-wri231 committed:
Fixed: role permission active all logic

themodernpk commented on Vue/vaahtwo/pages/public/setup/intall/Configuration.vue:
```diff
@@ -93,11 +93,13 @@ onMounted(async () => {
                       <div class="p-input">
                           <InputText v-model="store.config.env.app_name"
-                                     placeholder="App/Website Name"
+                                     placeholder="VaahCMS"
```
@we-vikram-wri231 the place should remain: `Enter your website name or app name`, and a helper message and validation for `Only Alpha Numeric` value are allowed

we-vikram-wri231 committed:
Updated: validation and placeholder text

we-vikram-wri231 committed:
Updated: validation and placeholder text

we-vikram-wri231 committed:
Updated: validation and placeholder text

we-vikram-wri231 committed:
Updated: validation and placeholder text

we-vikram-wri231 committed:
Updated: rebuild assets

we-vikram-wri231 committed:
Updated: rebuild assets

we-vikram-wri231 committed:
Updated: rebuild assets

we-vikram-wri231 committed:
Updated: rebuild assets

we-alam-a001 committed:
Merge remote-tracking branch 'origin/2.x-feature/update-users-localization' into 2.x-feature/update-users-localization

we-alam-a001 committed:
Updated: conditional if for language string

we-alam-a001 committed:
Updated: users store

we-alam-a001 committed:
Updated: localizations search placeholder

we-alam-a001 committed:
Updated: localizations search placeholder

we-alam-a001 committed:
Merge remote-tracking branch 'origin/2.x-feature/update-users-localization' into 2.x-feature/update-users-localization

we-alam-a001 committed:
Updated: users store

we-alam-a001 committed:
Updated: conditional if for language string

we-vikram-wri231 committed:
Updated: form button position

we-alam-a001 committed:
Updated: view role condition

we-alam-a001 committed:
Merge remote-tracking branch 'origin/2.x-feature/update-users-localization' into 2.x-feature/update-users-localization

we-alam-a001 committed:
Updated: language string of label

we-alam-a001 committed:
Updated: language string of label

we-vikram-wri231 committed:
Updated: avatar accessor

we-vikram-wri231 committed:
Updated: image url condition

themodernpk committed:
Merge pull request #218 from webreinvent/2.x-feature/update-users-localization

2.x feature -> 2.x develop | update users localization

themodernpk committed:
build: assets

themodernpk committed:
Merge branch '2.x-release/v2.2.3' into 2.x

themodernpk committed:
updated: changelog

themodernpk committed:
Merge branch '2.x' into 2.x-develop

we-vikram-wri231 committed:
Merge remote-tracking branch 'origin/2.x-feature/session-fixes' into 2.x-feature/session-fixes

we-alam-a001 committed:
Updated: localization of dashboard page

we-alam-a001 committed:
Updated: sidebar menu label extend and dashboard

we-alam-a001 committed:
Updated: sidebar menu label extend and dashboard

we-alam-a001 committed:
Added: conditional if for language string

we-alam-a001 committed:
Added: conditional if for language string

we-alam-a001 committed:
Fixed: login error while language is fr

we-alam-a001 committed:
Fixed: login error while language is fr

we-alam-a001 committed:
Updated: extend controller for label

we-alam-a001 committed:
Updated: extend controller for label

we-alam-a001 committed:
Fixed: to show bold text

we-alam-a001 committed:
Fixed: to show bold text

we-alam-a001 committed:
Updated: sidebar menu label extend and dashboard 


Added: conditional if for language string


Fixed: login error while language is fr


Updated: extend controller for label


Fixed: to show bold text

we-alam-a001 committed:
Merge remote-tracking branch 'origin/2.x-feature/update-dashbaord-localization' into 2.x-feature/update-dashbaord-localization

we-alam-a001 committed:
Updated: dashboard language string

we-alam-a001 committed:
Updated: dashboard language string

we-alam-a001 committed:
Removed: unused line

we-alam-a001 committed:
Removed: unused line

we-vikram-wri231 committed:
Merge remote-tracking branch 'origin/2.x-feature/added-restricted-actions-users' into 2.x-feature/added-restricted-actions-users

we-vikram-wri231 committed:
Updated: view role bulk function

we-vikram-wri231 committed:
Updated: bulk all condition

we-vikram-wri231 committed:
Updated: roles bulk actions

we-alam-a001 committed:
Added: functions to get language string

we-alam-a001 committed:
Added: functions to get language string

we-alam-a001 committed:
Added: run migrations and seeders after installed

we-alam-a001 committed:
Added: run migrations and seeders after installed

we-alam-a001 committed:
Updated: setup stores

we-alam-a001 committed:
Updated: setup stores

we-alam-a001 committed:
Updated: run seeders method

we-alam-a001 committed:
Updated: run seeders method

we-alam-a001 committed:
Updated: run migrations and seeders after installed

we-alam-a001 committed:
Updated: run migrations and seeders after installed

we-alam-a001 committed:
Updated: function in setup store

we-alam-a001 committed:
Updated: function in setup store

we-alam-a001 committed:
Added: run seeder functionality in localization

we-alam-a001 committed:
Added: run seeder functionality in localization

we-alam-a001 committed:
Added: functions to get language string


Added: run migrations and seeders after installed


Updated: setup stores


Updated: run seeders method


Updated: run migrations and seeders after installed


Updated: function in setup store


Added: run seeder functionality in localization

we-alam-a001 committed:
Merge remote-tracking branch 'origin/2.x-feature/generate-language-file-option' into 2.x-feature/generate-language-file-option

we-alam-a001 committed:
Updated: naming of language strings

we-alam-a001 committed:
Updated: naming of language strings

we-alam-a001 committed:
Updated: naming convention

we-alam-a001 committed:
Updated: naming convention

we-alam-a001 committed:
Updated: if condition in users actions

we-alam-a001 committed:
Updated: if condition in users actions

we-alam-a001 committed:
Updated: if condition in users form and view

we-alam-a001 committed:
Updated: if condition in users form and view

we-alam-a001 committed:
Updated: users assets language string

we-alam-a001 committed:
Updated: users assets language string

we-alam-a001 committed:
Updated: conditions for language string error

we-alam-a001 committed:
Updated: conditions for language string error

we-alam-a001 committed:
Updated: label of extend

we-alam-a001 committed:
Updated: label of extend

we-alam-a001 committed:
Updated: condition when no strings provided

we-alam-a001 committed:
Updated: condition when no strings provided

we-alam-a001 committed:
Added: watch for looking store assets loaded

we-alam-a001 committed:
Added: watch for looking store assets loaded

we-alam-a001 committed:
Updated: users title

we-alam-a001 committed:
Updated: users title

themodernpk committed:
Merge pull request #241 from webreinvent/2.x-feature/added-restricted-actions-users

2.x feature -> 2.x develop | added restricted actions users

we-alam-a001 committed:
Updated: dashboard language strings

we-alam-a001 committed:
Updated: dashboard language strings

we-alam-a001 committed:
Fixed: registration not found error message

we-alam-a001 committed:
Fixed: registration not found error message

we-vikram-wri231 committed:
Merge remote-tracking branch 'origin/2.x-feature/generate-language-file-option' into 2.x-feature/generate-language-file-option

we-alam-a001 committed:
Updated: page title strings

we-alam-a001 committed:
Merge remote-tracking branch 'origin/2.x-feature/generate-language-file-option' into 2.x-feature/generate-language-file-option

we-vikram-wri231 committed:
Updated: style overflow

themodernpk committed:
Merge pull request #244 from webreinvent/2.x-feature/fixx-notification-ui

2.x feature -> 2.x develop | Updated: style overflow

themodernpk committed:
Merge pull request #243 from webreinvent/2.x-feature/generate-language-file-option

2.x feature -> 2.x develop | Added: seeds & migrations and updated language strings

we-alam-a001 committed:
Merge remote-tracking branch 'origin/2.x-feature/update-dashbaord-localization' into 2.x-feature/update-dashbaord-localization

we-alam-a001 committed:
Updated: language strings after rebase

themodernpk committed:
Merge pull request #239 from webreinvent/2.x-feature/update-dashbaord-localization

2.x feature -> 2.x develop | update dashboard localization

we-vikram-wri231 committed:
Merge remote-tracking branch 'origin/2.x-feature/session-fixes' into 2.x-feature/session-fixes
